app/database/seeds/data/user_answer.json

```diff
@@ -422,5 +422,5061 @@
         "location_id": "00000000-0000-0000-0000-000000000040",
         "question_id": 100,
         "when_submitted": "2017-04-01 00:00:00"
+    },
+    {
+        "answer_value": 1,
```

Comment:
Test that this data can be synced.  We want the new user_answer values to get into app.accesslocator.com's database and eventually to demo.accesslocator.com's database.

Reply:
done